access_error.go

```diff
@@ -38,12 +38,16 @@ func (f *Fosite) writeJsonError(rw http.ResponseWriter, err error) {
 
 	rfcerr := ErrorToRFC6749Error(err)
 	if !f.SendDebugMessagesToClients {
-		rfcerr.Debug = ""
+		rfcerr = rfcerr.Sanitize()
 	}
 
-	js, err := json.Marshal(rfcerr)
+	js, err := json.MarshalIndent(rfcerr, "", "\t")
```

Comment:
I use `MarshalIndent` here so that both authorize error and access error are represented in the same way.

Reply:
I would expect JSON APIs to respond with machine readable code (minimized) as opposed to human readable (indented, formatted). Any particular reason why we would respond with human readable formats here?

Reply:
It was like that in the authorize error handler already. I just unified it to be the same. I do not really care (except that it will be a bit more work for me to change it) but I just wanted to be uniform. maybe `MarshalIndent` was used for errors so that it is easier to debug when manually inspecting the response when debugging.

Reply:
In that case I would prefer:

```suggestion
	js, err := json.Marshal(rfcerr)
```